migrations/20220810211254_drop_column_and_fks.up.fizz

```diff
@@ -0,0 +1,3 @@
+drop_foreign_key("requirement_types", "requirement_types_department_id_fkey", {"if_exists": true})
+change_column("requirement_types", "department_id", "uuid", {"null": true})
+drop_column("requirement_types", "department_id")
```

Comment:
It looks we also need to update the "requirement_types" rows by setting the "department_id" with null values to be able to remove this column.